modules/cnf-gathering-data-about-cluster-using-must-gather.adoc

```diff
@@ -0,0 +1,45 @@
+// Module included in the following assemblies:
+// Epic CNF-792 (4.8)
+// * scalability_and_performance/cnf-create-performance-profiles.adoc
+
+[id="gathering-data-about-your-cluster-using-must-gather_{context}"]
+= Gathering data about your cluster using `must-gather`
+
+The Performance Profile Creator (PPC) tool requires `must-gather` data. As a cluster administrator, run `must-gather` to capture information about your cluster .
+
+.Prerequisites
+
+* Access to the cluster as a user with the `cluster-admin` role.
+* Access to the Performance Addon Operator image.
+* The OpenShift Container Platform CLI (`oc`) installed.
+
+.Procedure
+
+. Navigate to the directory where you want to store the `must-gather` data.
+
+. Run `must-gather` on your cluster:
++
+[source,terminal]
+
+----
+$ oc adm must-gather --image=<PAO image> --dest-dir=<dir>
```

Comment:
Should be an underscore separating multiple words in replaceable values, so "<PAO_image>"